src/router/index.js

```diff
@@ -7,6 +7,7 @@ import { wallets } from '../services';
 Vue.use(Router);
 
 export default new Router({
+  mode: 'history',
```

Comment:
Need to remove this. Doesn’t work week with the way we are hosting the static assets.

Reply:
I made the change and adjusted navigating to the login screen.